packages/components/src/components/button-tile/button-tile-utils.ts

```diff
@@ -0,0 +1,16 @@
+import type { Theme } from '../../utils/theme';
+import type { ButtonAriaAttribute, ButtonType, ButtonVariant, LinkButtonIconName } from '../../types';
+
+export type ButtonTileAriaAttribute = ButtonAriaAttribute;
+export type ButtonTileIcon = LinkButtonIconName;
+export type ButtonTileType = ButtonType;
+
+export type ButtonProps = {
```

Comment:
Hmm instead of defining this from scratch can't we have this based on what `Button` and `Button Pure` are offering? Similar to what we did with typing the shared propTypes.

Reply:
Actually this type is not defined within the button component, so unfortunately there is nothing to reuse. The type here is only needed for react 
![image](https://user-images.githubusercontent.com/109523972/225310853-fd218caf-cfb2-4add-b647-6016c695af3b.png)

Reply:
Not sure if we are talking about the same thing.  
This type is used for typing props that are then used on a `p-button` or `p-button-pure`.  So why do we have to define this type manually and can't have it based on what is available from `p-button` or `p-button-pure` by using `InstanceType` or something?